qemu/tests/cfg/blockdev_inc_backup_add_persistent_bitmap_when_paused.cfg

```diff
@@ -5,7 +5,7 @@
 
 - blockdev_inc_backup_add_persistent_bitmap_when_paused:
     only Linux
-    only filesystem iscsi_direct ceph gluster_direct
+    only filesystem iscsi_direct gluster_direct
```

Comment:
remove ceph because of the product issue for qcow2 image on ceph?
do we want to make a comment here, like FIXME: add ceph back when qcow2 image on ceph issue is fixed, for we might forget this case

Reply:
Have updated, thanks zhenchao.